8.5-jre8/server.xml

```diff
@@ -71,7 +71,11 @@
     <Connector port="${port.http}" protocol="HTTP/1.1"
                connectionTimeout="20000"
                redirectPort="8443"
-               maxHttpHeaderSize="16384" />
+               maxHttpHeaderSize="16384"
+             compression="on"
+             compressionMinSize="1024"
+             noCompressionUserAgents="gozilla, traviata"
+             compressableMimeType="text/html,text/xml" />
```

Comment:
As per the latest Apache Tomcat reference this should be `compressibleMimeType` (versus `compressAbleMimeType`). Looks like both work but at some point it was renamed. Better to have it that way. Also, is there a reason for changing the default value of `compressibleMimeType`? Can we just leave it unchanged (keep it default)?